changelog/next_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+VERSION=$(cat "${1:-$DIR/../../dmd/VERSION}")
+# v2.076.1-b1 -> 2.076.1
+VERSION=${VERSION:1:7}
+# 2.076.1 -> (2 076 1)
+PARTS=(${VERSION//./ })
+# use 10#076 prefix to read octal as base10 int
+PARTS[1]=0$((10#${PARTS[1]} + 1))
+PARTS[2]=0
+# 2 077 0 -> 2.077.0
+echo "${PARTS[0]}.${PARTS[1]}.${PARTS[2]}"
```

Comment:
A check for a correct version would be great here, e.g. make sure VERSION matches a regex.

Reply:
We can even do better - I added a small testsuite.